controllers/ingressnodefirewallconfig_controller.go

```diff
@@ -106,6 +106,8 @@ func (r *IngressNodeFirewallConfigReconciler) syncIngressNodeFwConfigResources(c
 
 	data.Data["Image"] = os.Getenv("DAEMONSET_IMAGE")
 	data.Data["NameSpace"] = r.Namespace
+	data.Data["RBACProxyImage"] = os.Getenv("KUBE_RBAC_PROXY_IMAGE")
+	data.Data["PollPeriodSeconds"] = os.Getenv("POLL_PERIOD_SECONDS")
```

Comment:
this might be too much for Pollperiod what do u think ? just envvar in ds will be enough IMO